### PR TITLE
Global warhead totals area chart

### PR DESCRIPTION
**Pull request by urchykli**
I created a new sheet and imported the data because there is hidden data on the original sheet, but please let me know if you'd like me to do this in a different way.

### CODE REVIEW

**Comment by urchykli** [comment]
Also, I wasn't sure about the y axis labels, should they be 10k or 10,000?

**Comment by jnschrag** [comment]
@urchykli Sorry, I missed this when I was looking today, but can you update the folder so it's in `nuclear-capabilities`. Otherwise it's great, thanks!

**Comment by urchykli** [comment]
Both the area and bar charts are now ready for review.  Thanks!

File: nuclear-capabilities/global-nuclear-warheads/index.html [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  <!-- Include Highcharts Libraries; Adjust per project -->
+  <script src="https://code.highcharts.com/highcharts.js"></script>
+  <script src="https://code.highcharts.com/highcharts-more.js"></script>
+  <script src="https://code.highcharts.com/modules/data.js"></script>
+  <script src="https://code.highcharts.com/maps/modules/exporting.js"></script>
+  <!-- Include iFrame Resizer -->
+  <script src="https://csis-ilab.github.io/cpower-viz/globaljs/iframeSizer.contentWindow.min.js"></script>
+  <!-- Include China Power Highcharts Theme -->
+  <script src="https://csis-ilab.github.io/cpower-viz/globaljs/hc-cpp-basic-theme.js"></script>
+  <script src="../../globaljs/hc-cpp-basic-theme.js"></script>
```
Comment (jnschrag) [inline nuclear-capabilities/global-nuclear-warheads/index.html]: This line should be removed. In production, only the hosted version (https://csis-ilab.github.io) of the theme should be included.

File: nuclear-capabilities/strategic-nonstrategic-weapons/index.html [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  <!-- Include Highcharts Libraries; Adjust per project -->
+  <script src="https://code.highcharts.com/highcharts.js"></script>
+  <script src="https://code.highcharts.com/highcharts-more.js"></script>
+  <script src="https://code.highcharts.com/modules/data.js"></script>
+  <script src="https://code.highcharts.com/maps/modules/exporting.js"></script>
+  <!-- Include iFrame Resizer -->
+  <script src="https://csis-ilab.github.io/cpower-viz/globaljs/iframeSizer.contentWindow.min.js"></script>
+  <!-- Include China Power Highcharts Theme -->
+  <!-- <script src="https://csis-ilab.github.io/cpower-viz/globaljs/hc-cpp-basic-theme.js"></script> -->
+  <script src="../../globaljs/hc-cpp-basic-theme.js"></script>
```
Comment (jnschrag) [inline nuclear-capabilities/strategic-nonstrategic-weapons/index.html]: See note above about the themes.

**Pull request merged**